plugins/catalog/src/alpha.ts

```diff
@@ -15,3 +15,40 @@
  */
 
 export * from './translation';
+
+import {
+  DiscoveryApi,
+  FetchApi,
+  discoveryApiRef,
+  fetchApiRef,
+} from '@backstage/core-plugin-api';
+import { CatalogClient } from '@backstage/catalog-client';
+import {
+  createApiExtension,
+  createPlugin,
+} from '@backstage/frontend-plugin-api';
+import { catalogApiRef } from '@backstage/plugin-catalog-react';
+
+/** @alpha */
+export const CatalogApi = createApiExtension({
+  factory: {
+    api: catalogApiRef,
+    deps: {
+      discoveryApi: discoveryApiRef,
+      fetchApi: fetchApiRef,
+    },
+    factory: ({
+      discoveryApi,
+      fetchApi,
+    }: {
+      discoveryApi: DiscoveryApi;
+      fetchApi: FetchApi;
+    }) => new CatalogClient({ discoveryApi, fetchApi }),
+  },
+});
+
+/** @alpha */
+export default createPlugin({
+  id: 'plugin.catalog',
```

Comment:
Just `'catalog'` here

Reply:
Thanks for pointing it out, fixed here: https://github.com/backstage/backstage/pull/20169/commits/456c6311cc23609e3760bc2b39c5a7e01ba9f0bb